lessons/Early-Math/multiplicative-inverse/langs/en-us/pages/multiplicative-inverse.adoc

```diff
@@ -0,0 +1,83 @@
+= The Multiplicative Inverse Property
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+td > .content > .paragraph > * { vertical-align: middle; }
+</style>
+++++
+
+On the line next to each value, give the multiplicative inverse.
+
+[.FillVerticalSpace, cols="^.^3,^.^3,^.^3,^.^3,^.^3", stripes="none"]
+|===
+
+| @math{\frac{4}{5} } @fitb{5em}
```

Comment:
Why not make these look like the table below, and even _fuse_ the two top tables entirely? Everything should just be `A * B = 1`.